README.md

```diff
@@ -180,9 +180,8 @@ describing the feature/topic you are going to implement.
 
 # Contact
 
-- [@roman-khimov](https://github.com/roman-khimov) on GitHub
 - [@AnnaShaleva](https://github.com/AnnaShaleva) on GitHub
-- [@fyrchik](https://github.com/fyrchik) on GitHub
```

Comment:
Maybe we can just replace it by @fyfyrchik?

Reply:
I doubt it's useful, after all this README sets some expectations for people reading it. What can they expect of @fyrchik? What can they expect of @fyfyrchik? They can expect @AnnaShaleva to solve their question, that's what they should be aware of.